mava/configs/env/smac_gym.yaml

```
@@ -0,0 +1,25 @@
+# ---Environment Configs---
+defaults:
+  - _self_
+
+env_name: Starcraft  # Used for logging purposes.
```

Just to line up with literature. Since we are not really training on StarCraft II but SMAC which is a lot simpler. 
```suggestion
env_name: SMAC  # Used for logging purposes.
```

---

Also, can we please support the original SMAC as well?

---

I just 1 otherwise it will be confusing, so original SMAC if @Louay-Ben-nessir can get it to work

---

it's on my to-do list but it might take a while 😅